src/back/TicketManager/tests/Core/TicketManager.Core.ServicesTests/Endpoints/Accounts/PasswordValidatorTests.cs

```diff
@@ -0,0 +1,59 @@
+using TicketManager.Core.Services.Endpoints.Accounts;
+using TicketManager.Core.ServicesTests.Helpers;
+using Xunit;
+
+namespace TicketManager.Core.ServicesTests.Endpoints.Accounts;
+
+public class PasswordValidatorTests
+{
+    private const int TooShortErrorCode = 1;
+    private const int TooLongErrorCode = 2;
+    private const int InvalidErrorCode = 3;
+    private readonly PasswordValidator validator = new(TooShortErrorCode, TooLongErrorCode, InvalidErrorCode);
+    
+    [Theory]
+    [InlineData("")]
+    [InlineData("asdf")]
+    [InlineData("asdfasd")]
+    public void WhenTooShortPasswordIsProvided_ItShouldReturnFalseWithPasswordIsTooShortErrorCode(string password)
```

Comment:
You should probably make those passwords too short, but still valid. Currently you assume that length is tested first, but if `PasswordValidator` checked for presence of digits first this unit test would fail. Same for too long unit tests

Reply:
In default configuration that I haven't changed fluent validation returns all errors it finds. So `InvalidPassword` is returned as well and order doesn't matter.

Reply:
We are definitely squashing them. Keep information about each commit in commit description

Reply:
Fine, then squash them and we can merge